plugins/remotemetrics/sync.go

```diff
@@ -11,20 +11,23 @@ var isTangleTimeSynced atomic.Bool
 
 func checkSynced() {
 	oldTangleTimeSynced := isTangleTimeSynced.Load()
-	tts := deps.Tangle.TimeManager.Synced()
```

Comment:
Actually we'd be more interested in nodes running out of sync as in RCTT being different than CTT.

Reply:
Can you explain a bit better what you want to change here?

Reply:
Done